tsconfig.spec.json

```diff
@@ -4,6 +4,7 @@
     "outDir": "./out-tsc/spec",
     "resolveJsonModule": true,
     "esModuleInterop": true,
+    "sourceMap": true,
```

Comment:
Strictly speaking this shouldn't be necessary because it inherits it from `tsconfig.json`.